common/src/contract.ts

```diff
@@ -252,6 +259,13 @@ export type Stonk = {
   initialProbability: number
 }
 
+export type BountiedQuestion = {
+  outcomeType: 'BOUNTIED_QUESTION'
+  totalBounty: number
+  bountyPaid: number
+  bountyTxns: string[]
```

Comment:
Are these the id's of the txns? Should leave a comment at minimum

Reply:
oh yeah that was the idea, will do